msvc/libgc.targets

```diff
@@ -0,0 +1,181 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="libmonogc_la_sources">
+    <ClCompile Include="$(MonoSourceLocation)\libgc\allchblk.c">
+		  <ExcludedFromBuild Condition="'$(MONO_TARGET_GC)'!='boehm'">true</ExcludedFromBuild>
+	  </ClCompile>
+    <ClCompile Include="$(MonoSourceLocation)\libgc\alloc.c">
+		  <ExcludedFromBuild Condition="'$(MONO_TARGET_GC)'!='boehm'">true</ExcludedFromBuild>
```

Comment:
I think we could make this nicer to read if we had a property `TargetsSGenGC` or something and then just use:

```
<ExcludedFromBuild>$(TargetsSGenGC)</ExcludedFromBuild>
```

or can we exclude the whole ItemGroup?

Reply:
Great idea! I will apply that pattern, could probably be used in some other exclude from build cases as well.